### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by yuhaoliu7456** [ZhanzhouFeng/Pytorch-Implement-Faster-High-Res-Neural-Inpainting#2]
Why do you use the bce-loss in your implementation？
